.vscode/launch.json

```diff
@@ -0,0 +1,32 @@
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "type": "node",
+      "request": "launch",
+      "name": "Marktext: Main",
+      "program": "${workspaceFolder}/.electron-vue/dev-runner.js"
+    },
+    {
+      "name": "Marktext: Renderer",
+      "type": "chrome",
+      "request": "attach",
+      "port": 8315,
+      "webRoot": "${workspaceFolder}/src",
+      "timeout": 100000,
```

Comment:
15s (`15000`) or 30s (`30000`) is a good timeout.

Reply:
I prefer 30s.